core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -851,7 +892,7 @@ private void initializeTriggerables(TreeReference rootRef) {
             }
         }
 
-        evaluateTriggerables(applicable, rootRef);
+        evaluateTriggerables(applicable, rootRef, true);
```

Comment:
This method gets called in contexts which aren't repeats, so either the name of this true arg (isRepeatEntryInit) isn't correct, or this value isn't